cli/chaosli/cmd/context.go

```diff
@@ -0,0 +1,393 @@
+			percentCollectCondition[string(status.Type)+"/"+string(status.Status)] += 1.0 / float64(len(targetsInfo))
+		}
+		percentCollectPhase[string(pod.Status.Phase)] += 1.0 / float64(len(targetsInfo))
+
+		if i < MAXTARGETSHOW {
+			fmt.Printf(info + "\n\n")
+		}
+	}
+	PrintSeparator()
+	percentInfo := "Lets look at the overall status...\n"
+
+	for key, value := range percentCollectPhase {
+		roundedValue := value * 100.00
+		percentInfo += "\tPhase:                 " + key + "\n" +
+			"\tPercent:               " + fmt.Sprint(math.Round(roundedValue*100)/100) + "%\n\n"
+	}
+
+	for key, value := range percentCollectCondition {
+		roundedValue := value * 100.00
+		percentInfo += "\tCondition Type/Status: " + key + "\n" +
+			"\tPercent:               " + fmt.Sprint(math.Round(roundedValue*100)/100) + "%\n\n"
+	}
+	fmt.Println(percentInfo)
+
+	PrintSeparator()
+}
+
+func printNodeStatus(targetsInfo []v1.Node) {
+	percentCollectCondition := make(map[string]float64)
+
+	fmt.Println("\nLets take a look at the status of your Targeted Nodes...")
+
+	for i := 0; i < len(targetsInfo); i++ {
+		node := targetsInfo[i]
+		info := "\t🥸  Node Name: " + node.Name + "\n" +
+			"\t📲 Node Addresses: \n"
+
+		for _, address := range node.Status.Addresses {
+			info += "\t\t⭕️ Type: " + string(address.Type) + "\n" +
+				"\t\tℹ️  Address: " + address.Address + "\n"
+		}
+		info += "\t📜 Node Conditions:\n"
+
+		for _, status := range node.Status.Conditions {
+			info = info +
+				"\t\t⭕️ Type: " + string(status.Type) + "\n" +
+				"\t\tℹ️  Status: " + string(status.Status) + "\n" +
+				"\t\t🤨 Reason: " + status.Reason + "\n\n"
+			percentCollectCondition[string(status.Type)+"/"+string(status.Status)] += 1.0 / float64(len(targetsInfo))
+		}
+
+		if i < MAXTARGETSHOW {
+			fmt.Printf(info + "\n\n")
+		}
+	}
+
+	PrintSeparator()
+
+	percentInfo := "Lets look at the overall status...\n"
+	for key, value := range percentCollectCondition {
+		roundedValue := value * 100.00
+		percentInfo += "\tCondition Type/Status: " + key + "\n" +
+			"\tPercent:               " + fmt.Sprint(math.Round(roundedValue*100)/100) + "%\n\n"
+	}
+
+	fmt.Println(percentInfo)
+	PrintSeparator()
+}
+
+func checkKubectl() error {
+	cmd := exec.Command("kubectl", "get", "pods", "-n", "chaos-engineering")
+	_, err := cmd.Output()
+
+	if err != nil {
+		return err
+	}
+
+	return nil
+}
+
+var contextCmd = &cobra.Command{
+	Use:   "context",
+	Short: "contextualizes disruption config",
+	Long:  `makes use of kubectl to give a better idea of how big the scope of the disruption will be.`,
+	PreRunE: func(cmd *cobra.Command, args []string) error {
+		path, _ := cmd.Flags().GetString("path")
+		return validatePath(path)
+	},
+	Run: func(cmd *cobra.Command, args []string) {
+		path, _ := cmd.Flags().GetString("path")
+		contextualize(path)
+	},
+}
+
+func contextualize(path string) {
+	var disruption v1beta1.Disruption
+
+	fullPath, err := filepath.Abs(path)
```

Comment:
The "read file, unmarshal, validate" logic is now repeated thrice, consider extracting this into a function?